src/generated/resources/openapi.yaml

```diff
@@ -1023,6 +1047,10 @@ components:
           $ref: "#/components/schemas/JsonNode"
         value:
           $ref: "#/components/schemas/JsonNode"
+        key_schema:
+          $ref: "#/components/schemas/SchemaDetails"
+        value_schema:
+          $ref: "#/components/schemas/SchemaDetails"
```

Comment:
Since these keys might also hold information about keys/values without a schema, what about renaming them to `key_format` and `value_format`?